<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>319</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Memos</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListView" name="listMemos">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>0</y>
      <width>231</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QListView" name="listPeers">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRefreshPeers">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh List</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSendMemo">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Send Memo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRecordMemo">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Record</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPlayMemo">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDeleteMemo">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>319</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuClose_Memos">
    <property name="title">
     <string>Close Memos</string>
    </property>
   </widget>
   <addaction name="menuClose_Memos"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
